package.json

```diff
@@ -86,5 +86,8 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "engines": {
+    "node": "^14.17.0 || >=16.0.0"
```

Comment:
You'll need to `npm install` afterwards to ensure this makes it into the lockfile too. In other repos we've just done `>= 14.0` — not sure we need specific features introduced between 14.0 and 14.17?

Reply:
Nah, let's stick with what it is for consistency.. I think notifications I used >= 14, So I'll update there to match